aiodocker/services.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+
+class DockerServices(object):
+    def __init__(self, docker):
+        self.docker = docker
+
+    async def list(self, filters: dict={}) -> list:
+        """
+        Returns a list of services
+
+        Available filters:
+            id=<service id>
+            label=<service label>
+            mode=["replicated"|"global"]
+            name=<service name>
+
+        """
+        
+        response = await self.docker._query_json(
+            "services",
+            method='GET',
+            params=filters
+        )
+        return response
+
+    async def create(self, task_template, name=None,
+                     labels=None, mode=None,
+                     update_config=None, networks=None,
+                     endpoint_config=None, endpoint_spec=None) -> dict:
+        """
+        Creates a service
+        
+        Returns a dictionary with the ID of the service
+        """
+
+        config = {
+            "Name": name,
+            "Labels": labels,
+            'TaskTemplate': task_template,
+            'Mode': mode,
+            'Networks': networks,
+            'EndpointSpec': endpoint_spec
+        }
+
+        data = {}
+
+        if config is not None and isinstance(config, dict):
+            for k, v in config.items():
```

Comment:
minor thing, but what do you think about `{k: v for k, v in config.items() if v is not None}` instead for loop ?